astro/src/content/quickstarts/quickstart-python-django-web.mdx

```diff
@@ -0,0 +1,258 @@
+
+
+### Project Settings
+
+Overwrite your `mysite/settings.py` contents with the following code.
+
+`settings.py`
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-python-django-web/main/complete-application/mysite/mysite/settings.py" lang="python" />
+
+Here, your `app` and `mozilla_django_oidc` are added to the list of `INSTALLED_APPS`. The authentication library from Mozilla is set up by `mozilla_django_oidc` and the `OIDC_OP_` constants imported from the `.env` file. This automatically handles the interaction with FusionAuth on the endpoints configured in this file and the matching ones configured in the `kickstart.json` file that you used to initialize FusionAuth.
+
+### Project URLs
+
+Overwrite the `mysite/urls.py` file with the following code.
+
+`urls.py`
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-python-django-web/main/complete-application/mysite/mysite/urls.py"
+ lang="python" />
+
+ * The `admin` site is the default Django admin.
+ * The `app` URLs are for the app you created in your project.
+ * The `oidc` URLs are automatically generated from the settings you configured in the previous step.
+
+### App URLs
+
+From this point, you'll work in the `app` not in the main project. 
+
+Create an `app/urls.py` file and set its content to match the project URLs.
+
+`urls.py`
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-python-django-web/main/complete-application/mysite/app/urls.py"
+ lang="python" />
+
+### App Views
+
+Now overwrite the `app/views.py` file. There are three pages: the home page, the account page, and the make change page.
+
+`views.py`
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-python-django-web/main/complete-application/mysite/app/views.py"
+ lang="python" />
+
+Here's what these files do:
+
+* The `app` view, or home page, redirects the user to the `account` view if they are already logged in.
+* The `account` view checks whether the user is logged in. If the user is logged in, it returns the account page template, passing in the user's email address returned from FusionAuth. If the user isn't logged in, they are redirected to the login page. This also happens in the change page.
+* The `logout` view is necessary because you need to clear the session for the user in Django after the user logs out of FusionAuth. FusionAuth sends the user to this view after logging them out, which then returns them to the home page.
+* The `change` view looks longer only because it demonstrates how to do some calculations if the request is a POST instead of a GET. These calculations are then passed to the template in the `change` variable to display onscreen.
+
+## Customization
+
+The final step is to create the HTML pages and style them with CSS.
+
+* Copy the `static` folder with its contents from `complete-application/mysite/app/static` to your `/mysite/app/static` location.
```

Comment:
Please provide commands for me to copy. Maybe state where you are running the commands from.